### PR TITLE
regularize and consolidate night elf dynasties - DRAFT

### PR DESCRIPTION
**Pull request by edmund-h**
## Developer changelog:
- removed `dynasties/100000_night_elves.txt`
- moved all currently used night elf dynasties from that file to `dynasties/128000_night_elves.txt`
- updated non-numbered dynasty names to use the format `dynn_name` for clarity and ease of searching
- changed some dopey automatically generated nightelf names to cooler ones

### CODE REVIEW

**Comment by Grekopithikos** [comment]
Wouldn't ctrl + f `dynasty` do the same when searching?

**Comment by edmund-h** [comment]
closing this because i want to get to work on the night elf nobility titles

> Wouldn't ctrl + f `dynasty` do the same when searching?

it does not bring up the definition

File: common/culture/name_lists/wc_darnassian.txt [hunk 1]
```diff
@@ -77,7 +77,7 @@
 	male_names = {
 		Aelgorn Aldryth Allendril Altruis Andalar Andoril Anelas Anren Arantir Ardan Aren Arvell Ashelan Astalor Astran Athridas Auran Baelir Balthule  
 		Baritanas Broll Cerellean Corithras Delandros Deldren Dralith Elerion Elloric Eranril Erelas Erion Eriun Evenar Faeolas Falorn   
-		Falrin Farodin Feronas Ferryn Filandras Firondren Foriel Fyldan Gaerolas Galind Gilshalan Golras Halsteth Idras Isoraen Ithalaine Ithis
```
Comment (Grekopithikos) [inline common/culture/name_lists/wc_darnassian.txt]: Ithis is a night elf name
https://warcraft.wiki.gg/wiki/Ithis_Moonwarden

File: history/characters/340000_night_elf.txt [hunk 2]
```diff
@@ -2155,13 +2155,14 @@ arantir={
...
-	name = Wifinnyia
+#dynasty = dynn_swiftarrow
+night_elf1 = { # Warsong Gulch Battlemaster
+	name = Su'ura
 	female = yes
-	dynasty = night_elf1
+	dynasty = dynn_swiftarrow # note - reuse autumnshade somewhere
 	religion = kaldorei_religion
 	culture = night_elf
+	trait=education_martial_4 trait=
```
Comment (Grekopithikos) [inline history/characters/340000_night_elf.txt]: "trait="

File: common/culture/name_lists/wc_darnassian.txt (hunk 1 above)
Comment (edmund-h) [reply common/culture/name_lists/wc_darnassian.txt]: ok will fix this

File: history/characters/340000_night_elf.txt (hunk 2 above)
Comment (edmund-h) [reply history/characters/340000_night_elf.txt]: fuuuuck ok will finish her

**Comment by edmund-h** [comment]
changed to draft because i am going to redo the male night elf names and possibly the high elf ones as well.

File: history/characters/340000_night_elf.txt (hunk 2 above)
Comment (edmund-h) [reply history/characters/340000_night_elf.txt]: ok fixed apparently this character was already in game

File: common/culture/name_lists/wc_darnassian.txt (hunk 1 above)
Comment (edmund-h) [reply common/culture/name_lists/wc_darnassian.txt]: fixed

**Pull request closed without merging**